docs/docs-content/clusters/public-cloud/azure/azure-cloud.md

```diff
@@ -47,6 +47,14 @@ Use the following steps to add an Azure or Azure Government account in Palette o
 | **Disable Properties**            | This option prevents Palette and VerteX from creating Azure Virtual Networks (VNets) and other network resources on your behalf for static placement deployments. If you enable this option, all users must manually specify a pre-existing VNet, subnets, and security groups when creating clusters. |
 | **Connect Private Cloud Gateway** | Select this option to connect to a Private Cloud Gateway (PCG) if you have a PCG deployed in your environment. Refer to the PCG [Architecture](../../pcg/architecture.md) page to learn more about a PCG.                                                                                              |
 
+:::info
```

Comment:
This admonition and the table above it are not properly aligned with step five 👁️

Reply:
Eagle eye 🦅